### DISCUSSION TITLE
esiste sandbox? esempi con chiamate in curl ?

### DISCUSSION THREAD

**Discussion by faustf** [fattureincloud/api#392] [Q&A] [answered]
Buon giorno a tutti , come da  titolo volevo sapere 
1. se esiste sandbox  per fare test e sviluppare , e se si  e' possibile avere le api per  programmare  anche se uno  non e' cliente  incloud? 
2. ci sono degli esempi di chiamate API fatte direttamente  con curl  ? grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @faustf 
può leggere questa guida: [Developer Account](https://developers.fattureincloud.it/docs/developer-account)

Per ottenere gli esempi con cURL, suggeriamo di usare la nostra [API Reference](https://developers.fattureincloud.it/docs/api-reference) oppure la [Postman Collection](https://developers.fattureincloud.it/docs/quickstarts/postman-collection) per testare i nostri esempi; entrambi gli strumenti permettono poi di ottenere la richiesta sotto forma di cURL (oppure, nel caso di Postman, anche come codice in diversi linguaggi e con diverse librerie, ad esempio PHP cURL).

Suggeriamo comunque di prendere in considerazione l'utilizzo delle nostre [SDK](https://developers.fattureincloud.it/docs/sdks/), in quanto permettono di velocizzare lo sviluppo minimizzando gli errori.

Qui sotto uno screenshot della funzionalità CodeSnippet di Postman
![image](https://github.com/fattureincloud/api/assets/7254293/deb4a7ce-9b9b-4cad-93f8-1803c627dced)

A disposizione
Mauro

**Reply by faustf** [reply]
Grazie  gentilissimo

**Reply by valmoz** [reply]
Grazie a lei, buona giornata
